contractcourt/channel_arbitrator.go

```diff
@@ -354,6 +359,10 @@ func (c *ChannelArbitrator) Start() error {
 		}),
 	)
 
+	// Retrieve relay fee for dust limit calculation. Assume that this will
+	// not change from here on.
```

Comment:
is this a safe assumption?

Reply:
I guess you could restart the backend with a different fee rate. Sweeper already makes this assumption though. Fixing that doesn't seem high prio to me. But I reworked this commit to obtain the relay fee rate from the sweeper. At least make the assumption only in one place.